Makefile

```diff
@@ -302,6 +303,7 @@ bundle-hub: manifests kustomize operator-sdk ## Generate hub bundle manifests an
 		$(KUSTOMIZE) edit add patch --name ramen-hub-operator.v0.0.0 --kind ClusterServiceVersion\
 		--patch '[{"op": "replace", "path": "/spec/replaces", "value": "$(REPLACES)"}]'
 	$(SED_CMD) -e "s,ramenOpsNamespace: ramen-ops,ramenOpsNamespace: $(RAMEN_OPS_NAMESPACE)," -i config/hub/manager/ramen_manager_config.yaml
+	$(SED_CMD) -e "s,veleroNamespaceName: velero,veleroNamespaceName: $(VELERO_NAMESPACE)," -i config/hub/manager/ramen_manager_config.yaml
```

Comment:
We already handle this in ramenctl, see:
https://github.com/RamenDR/ramen/blob/main/ramenctl/ramenctl/resources/configmap.yaml

If the defaults that works for upstream never reach downstream we can simplify ramnectl but adding all the defeaults ramen config.